hackathon_site/dashboard/frontend/src/App.tsx

```diff
@@ -59,6 +60,11 @@ const UnconnectedApp = () => {
                     <Route exact path="/login" component={Login} />
                     <Route exact path="/orders" component={withUserCheck(Orders)} />
                     <Route exact path="/teams" component={withUserCheck(Teams)} />
+                    <Route
+                        exact
+                        path="/team-detail"
+                        component={withUserCheck(TeamDetail)}
```

Comment:
Sorry about this Luke, I told you the wrong setup. could you change this url to "/teams/:id" so that every team has their own page.

To get the /:id value you can use the useParams hook
e.g. `const { id } = useParams()`
in the team detail page file